apps/examples/src/examples/state-store/StateStoreExample.tsx

```diff
@@ -0,0 +1,99 @@
+import { TLComponents, Tldraw, TLShape, track, useEditor, useQuickReactor, useValue } from 'tldraw'
+import 'tldraw/tldraw.css'
+
+// [1]
+const InfoPanel = track(() => {
+	const editor = useEditor()
+	const tool = editor.getCurrentToolId()
+	const zoom = editor.getZoomLevel().toFixed(2)
+	useQuickReactor(
+		'only red shapes',
+		() => {
+			const shapes = editor.getCurrentPageShapes()
+
+			editor.updateShapes(
+				shapes.map((shape: TLShape) => {
+					if ('color' in shape.props) {
+						return { ...shape, props: { ...shape.props, color: 'red' } }
+					} else {
+						return shape
+					}
+				})
+			)
+		},
+		[editor]
+	)
+	return (
+		<div style={{ pointerEvents: 'all', backgroundColor: 'thistle', fontSize: 14, padding: 8 }}>
+			<div>tool: {tool}</div>
+			<div>zoom: {zoom}</div>
+			<div style={{ padding: 8 }}>Only red shapes</div>
+		</div>
+	)
+})
+
+// [2]
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+function AlternativeInfoPanel() {
+	const editor = useEditor()
+	const tool = useValue(
+		'current tool',
+		() => {
+			if (!editor) throw new Error('No editor')
+			return `Current Tool: ${editor.getCurrentToolId()}`
+		},
+		[editor]
+	)
+	const zoom = useValue(
+		'zoom',
+		() => {
+			if (!editor) throw new Error('No editor')
+			return `Zoom Level: ${editor.getZoomLevel().toFixed(2)}`
+		},
+		[editor]
+	)
+
+	return (
+		<div style={{ pointerEvents: 'all', backgroundColor: 'thistle', fontSize: 14, padding: 8 }}>
+			<div>{tool}</div>
+			<div>{zoom}</div>
+		</div>
+	)
+}
+
+const components: TLComponents = {
+	SharePanel: InfoPanel,
+	// SharePanel: AlternativeInfoPanel,
+}
+
+export default function StateStoreExample() {
+	return (
+		<div className="tldraw__editor">
+			<Tldraw components={components} />
+		</div>
+	)
+}
+
+/* 
+
+Tldraw uses a signals library called Signia to manage its state and store. You can subscribe to
```

Comment:
this isn't really true any more
```suggestion
Tldraw uses a signals to manage its state and store. You can subscribe to
```